recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Support latest gtest
+    # https://github.com/msgpack/msgpack-c/commit/4a1db60af8d6bdb258a34352f5cd8d9800544372.patch
+    - gtest.patch
+    # This is needed for a bug in gcc version used where isnan is ambiguous. (::isnan and std::isnan)
+    - ambiguous_nan_inf.patch   # [linux]
```

Comment:
@isuruf the correct way of calling ``isinf()`` and ``isnan()`` in C++ is to include ``<cmath>`` and call ``std::isinf()`` and ``std::isnan()``. I suspect that the patch does not work in OSX because it is using libc++, which is not probably pulling in the ``<cmath>`` header through the inclusion of some other header.

Shall I give it a go? I cannot do it right now but I can update the PR tonight.